src/clostack/signature.clj

```diff
@@ -4,14 +4,37 @@
            javax.crypto.Mac
            java.util.Base64))
 
+
+(def byte-array?
+  (partial instance? (Class/forName "[B")))
+
+
```

Comment:
I've been thinking on dispatching with multimethods using `(mapv type args)` but it doesn't dispatch on nil values and takes more code.

@pyr what approach do you prefer?

Reply:
you can use `clojure.core/bytes?`